mne/preprocessing/artifact_detection.py

```diff
@@ -4,14 +4,74 @@
 
 
 import numpy as np
-
+from scipy.stats import zscore
+from scipy.ndimage.measurements import label
 from ..annotations import (Annotations, _annotations_starts_stops)
 from ..transforms import (quat_to_rot, _average_quats, _angle_between_quats,
                           apply_trans, _quat_to_affine)
+from ..filter import filter_data
 from .. import Transform
 from ..utils import (_mask_to_onsets_offsets, logger)
 
 
+def annotate_muscle(raw, threshold=1.5, picks=None, min_length_good=.1):
+    """Detect segments with muscle artifacts.
+
+    Detects segments periods that contains high frequency activity beyond the
+    specified threshold. Muscle artifacts are most notable in the range of 110-
+    140Hz.
+
+    Raw data is band pass filtered between 110 and 140 Hz, the signal envelope
+    computed, z-scored across samples, channel averaged and low-pass
+    filtered to smooth transient peaks.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        Data to compute head position.
+    threshold : float
+        The threshod for selecting segments with muscle activity artifacts.
```

Comment:
```suggestion
        The threshold for selecting segments with muscle activity artifacts.
```

Reply:
What are the units here? z score units might be a good default.